### PR TITLE
[fix](inverted index) Cloud mode supports Chinese tokenizer with default conversion to lowercase

### PR DESCRIPTION
**Pull request by zzzxl1993**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zzzxl1993** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/catalog/MaterializedIndexMeta.java [hunk 1]
```diff
@@ -361,4 +364,25 @@ public void initColumnNameMap() {
             definedNameToColumn.put(normalizeName(column.getDefineName()), column);
         }
     }
+
+    public boolean initWithExtraSchema() {
```
Comment (xiaokang) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/MaterializedIndexMeta.java:368]: move to Index class

File: fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java [hunk 2]
```diff
@@ -1551,6 +1551,9 @@ public void readFields(DataInput in) throws IOException {
             long indexId = in.readLong();
             this.indexNameToId.put(indexName, indexId);
             MaterializedIndexMeta indexMeta = MaterializedIndexMeta.read(in);
+            if (indexMeta.initWithExtraSchema()) {
```
Comment (xiaokang) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java:1554]: do not do it in readFields

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run p0

**Comment by zzzxl1993** [comment]
run p0

**Comment by zzzxl1993** [comment]
run p0

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run p0

**Comment by zzzxl1993** [comment]
run cloud_p1

**Comment by zzzxl1993** [comment]
run buildall

**Review by airborne12 — APPROVED** [review]
LGTM

**Review by xiaokang — APPROVED** [review]
LGTM

**Pull request merged**